platform/src/core/urls/deviceMonitoring.js

```diff
@@ -1,5 +1,7 @@
 import { stripTrailingSlash } from '../utils/strings';
 
-const BASE_DEVICE_MONITORING_URL = stripTrailingSlash(process.env.NEXT_PUBLIC_API_BASE_URL);
+const BASE_DEVICE_MONITORING_URL = stripTrailingSlash(
+  process.env.NEXT_PUBLIC_BASE_DEVICE_MONITORING_URL,
+);
 
-export const COLLOCATION = `${BASE_DEVICE_MONITORING_URL}/monitor/devices/collocation`;
+export const COLLOCATION = `${BASE_DEVICE_MONITORING_URL}/api/v1/monitor/devices/collocation`;
```

Comment:
Please note that we can also have `api/v1`  substring inside the `BASE_DEVICE_MONITORING_URL` variable. That way we can avoid hard coding the API version (given that API v1 is being phased out)

Reply:
Ok. Thanks.

Reply:
Thanks. I have updated on my end. Will push commit after finalising on changes